frontend/actions/review.ts

```diff
@@ -25,18 +25,56 @@ export function useGetReviewRequestsForUser(isActive = true) {
   }
 }
 
-export function useGetNumReviews() {
+export function useGetReviewRequestsForModel(modelId, semver?, isActive = true) {
   const { data, error, mutate } = useSWR<
     {
-      count: number
+      reviews: ReviewRequestInterface[]
     },
     ErrorInfo
-  >('/api/v2/reviews/count', fetcher)
+  >(
+    semver
+      ? `/api/v2/reviews?${qs.stringify({
+          active: isActive,
+          modelId,
+          semver,
+        })}`
+      : `/api/v2/reviews?${qs.stringify({
+          active: isActive,
+          modelId,
+        })}`,
+    fetcher,
+  )
+
+  return {
+    mutateReviews: mutate,
+    reviews: data ? data.reviews : [],
+    isReviewsLoading: !error && !data,
+    isReviewsError: error,
+  }
+}
+
+// TODO - this API has been removed
```

Comment:
This reviews endpoint now returns a header with the count. Recommend changing this to a `HEAD` request and returning the value in the header.